bower.json

```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "Chart.js": "chartjs#^1.0.2",
     "polymer": "Polymer/polymer#^1.1.0",
-    "iron-flex-layout": "PolymerElements/iron-flex-layout"
+    "iron-resizable-behavior": "PolymerElements/iron-resizable-behavior#~1.0.2"
```

Comment:
@Awarua- The API for `iron-resizable-behavior` is not very volatile, so maybe you should consider using `#^1.0.2` instead of `#~1.0.2` in lieu of maintaining compatibility with all other elements this library might be installed with, so __bower__ compatibility errors don't arise.